include/sched.h

```diff
@@ -110,4 +117,18 @@ static inline void execute_tasks(void) {
     run_tasks(get_bsp_cpu());
     wait_for_all_cpus();
 }
+
+static inline void set_task_repeat(task_t *task, task_repeat_t value) {
```

Comment:
Is the compiler going to complain with the right flags that `value` is outside the values specified in `task_repeat_t`?

Reply:
No, enum is of type unsigned int.